packages/@react-aria/utils/src/filterDOMProps.ts

```diff
@@ -31,7 +31,11 @@ interface Options {
 
 const propRe = /^(data-.*)$/;
 
-// Filters out all props that aren't valid DOM props or are user defined via override prop obj.
+/**
+ * Filters out all props that aren't valid DOM props or defined via override prop obj.
+ * @param props - props to filter for a component
```

Comment:
each of these descriptions for params should start with a capitalized letter and end with a period

Reply:
That does not seem correct for react-aria. Should I fix this in a different PR?

all32111-macOS:react-spectrum-v3 all32111$ git grep -n @param packages/\@react-aria
packages/@react-aria/button/src/useButton.ts:30: * @param props - props to be applied to the button
packages/@react-aria/button/src/useButton.ts:31: * @param ref - a ref to a DOM element for the button
packages/@react-aria/checkbox/src/useCheckbox.ts:27: * @param props - props for the checkbox
packages/@react-aria/checkbox/src/useCheckbox.ts:28: * @param state - state for the checkbox, as returned by `useToggleState`
packages/@react-aria/checkbox/src/useCheckbox.ts:29: * @param inputRef - a ref for the HTML input element
packages/@react-aria/i18n/src/useCollator.ts:20: * @param options - collator options
packages/@react-aria/i18n/src/useDateFormatter.ts:20: * @param options - formatting options
packages/@react-aria/i18n/src/useMessageFormatter.ts:29: * @param strings - a mapping of languages to strings by key
packages/@react-aria/i18n/src/useNumberFormatter.ts:20: * @param options - formatting options
packages/@react-aria/listbox/src/useListBox.ts:47: * @param props - props for the listbox
packages/@react-aria/listbox/src/useListBox.ts:48: * @param state - state for the listbox, as returned by `useListState`
packages/@react-aria/listbox/src/useListBoxSection.ts:37: * @param props - props for the section
packages/@react-aria/listbox/src/useOption.ts:63: * @param props - props for the option
packages/@react-aria/listbox/src/useOption.ts:64: * @param state - state for the listbox, as returned by `useListState`
packages/@react-aria/menu/src/useMenu.ts:39: * @param props - props for the menu
packages/@react-aria/menu/src/useMenu.ts:40: * @param state - state for the menu, as returned by `useListState`
packages/@react-aria/menu/src/useMenuItem.ts:66: * @param props - props for the item
packages/@react-aria/menu/src/useMenuItem.ts:67: * @param state - state for the menu, as returned by `useTreeState`
packages/@react-aria/menu/src/useMenuSection.ts:37: * @param props - props for the section
packages/@react-aria/menu/src/useMenuTrigger.ts:34: * @param props - props for the menu trigger
packages/@react-aria/menu/src/useMenuTrigger.ts:35: * @param state - state for the menu trigger
packages/@react-aria/radio/src/useRadio.ts:39: * @param props - props for the radio
packages/@react-aria/radio/src/useRadio.ts:40: * @param state - state for the radio group, as returned by `useRadioGroupState`
packages/@react-aria/radio/src/useRadio.ts:41: * @param ref - ref to the HTML input element
packages/@react-aria/radio/src/useRadioGroup.ts:32: * @param props - props for the radio group
packages/@react-aria/radio/src/useRadioGroup.ts:33: * @param state - state for the radio group, as returned by `useRadioGroupState`
packages/@react-aria/searchfield/src/useSearchField.ts:33: * @param props - props for the search field
packages/@react-aria/searchfield/src/useSearchField.ts:34: * @param state - state for the search field, as returned by `useSearchFieldState`
packages/@react-aria/searchfield/src/useSearchField.ts:35: * @param inputRef - a ref to the input element
packages/@react-aria/select/src/useSelect.ts:49: * @param props - props for the select
packages/@react-aria/select/src/useSelect.ts:50: * @param state - state for the select, as returned by `useListState`
packages/@react-aria/switch/src/useSwitch.ts:26: * @param props - props for the switch
packages/@react-aria/switch/src/useSwitch.ts:27: * @param state - state for the switch, as returned by `useToggleState`
packages/@react-aria/switch/src/useSwitch.ts:28: * @param ref - ref to the HTML input element
packages/@react-aria/textfield/src/useTextField.ts:28: * @param props - props for the text field
packages/@react-aria/textfield/src/useTextField.ts:29: * @param ref - ref to the HTML input element
packages/@react-aria/utils/src/filterDOMProps.ts:36: * @param props - props to filter for a component
packages/@react-aria/utils/src/filterDOMProps.ts:37: * @param opts - props to override
packages/@react-aria/utils/src/mergeProps.ts:25: * @param a - the first set of props to merge.
packages/@react-aria/utils/src/mergeProps.ts:26: * @param b - the second set of props to merge.
packages/@react-aria/utils/src/useId.ts:24: * @param defaultId - default component id
packages/@react-aria/utils/src/useLabels.ts:19: * @param props - aria label props
packages/@react-aria/utils/src/useLabels.ts:20: * @param defaultLabel - default value for aria-label when not present